###Service that sends the velocity to the tiago and returns the position

#estimated impact_velocity with what the cube should be hit
float32 impact_velocity
---
#returns the xyz position of the cube after being hit
geometry_msgs/Point position